lib/ActivityStreams.js

```diff
@@ -66,6 +66,11 @@ ActivityStreams.prototype = {
           this.send(am.actions.Response("RECENT_BOOKMARKS_RESPONSE", links), worker);
         });
         break;
+      case am.type("RECENT_LIKS_REQUEST"):
```

Comment:
but it obviously didn't. @k88hudson what was your intention behind the action manager?